openaz-xacml/src/main/java/org/apache/openaz/xacml/std/jaxp/JaxpResponse.java

```diff
@@ -102,18 +103,16 @@ public static JaxpResponse load(File fileXmlResponse) throws ParserConfiguration
             return null;
         }
 
-        NodeList nodeListRoot = document.getChildNodes();
-        if (nodeListRoot == null || nodeListRoot.getLength() == 0) {
-            logger.warn("No child elements of the XML document");
-            return null;
-        }
-        Node nodeRoot = nodeListRoot.item(0);
-        if (nodeRoot == null || nodeRoot.getNodeType() != Node.ELEMENT_NODE) {
-            logger.warn("Root of the document is not an ELEMENT");
-            return null;
-        }
+        Node nodeRoot = document.getDocumentElement();
 
-        JAXBContext context = JAXBContext.newInstance(ResponseType.class);
+        if(context == null) {
```

Comment:
dont need all this just do this

```
static {
        try {
            // one time instance creation
            jaxbContext = JAXBContext.newInstance(Request.class, Response.class);
        } catch (JAXBException e) {
            e.printStackTrace();
        }
    }

```
for both request and respone in a singleton class and reuse as required.